buildDoc.sh

```diff
@@ -12,9 +12,15 @@ $GOPATH/bin/swag --version
 $GOPATH/bin/swag init --parseDependency --generalInfo tide-whisperer.go --output docs
 
 # When tag is present, openapi doc is renamed before being deployed to S3
+# It is stored in a new directory that will be used as source by the Travis deploy step
 if [ -n "${TRAVIS_TAG:-}" ]; then
     APP="${TRAVIS_REPO_SLUG#*/}"
     APP_TAG="${APP}-${TRAVIS_TAG}"
     mkdir docs/openapi
     mv docs/swagger.json docs/openapi/${APP_TAG}-swagger.json
+    # If this is not a release candidate but a "true" release, we consider this doc is the latest
+    # we create a copy named "latest" to be consumed by documentation website using SwaggerUI
+    if [[ ! ${TRAVIS_TAG} =~ rc[0-9] ]]; then
```

Comment:
This pattern doesn't match an alpha or a beta release, we could match a real version like this: 
```suggestion
    # we create a copy named "latest" to be consumed by documentation website using SwaggerUI
    if [[  ${TRAVIS_TAG} =~ [0-9]+\.[0-9]+\.[0-9]+$ ]]; then
```

Reply:
@CedricPerotto ok but we may have problems when we have version 1.10.0 for example?